packages/terra-date-picker/src/terra-dev-site/doc/example/DatePickerField.jsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import classNames from 'classnames/bind';
 import PropTypes from 'prop-types';
-import DatePickerField from 'terra-date-picker';
+import DatePickerField from '../../../DatePickerField';
```

Comment:
```suggestion
import DatePickerField from 'terra-date-picker/lib/DatePickerField';
```
nit : I think we have decided to not have import with file-path on examples page.